app/static/js/volunteerProfile.js

```diff
@@ -0,0 +1,80 @@
+$(document).ready(function(){
+
+  $(".form-check-input").click(function updateInterest(){
+    var programID = $(this).attr('id');
+    var interest = $(this).is(':checked');
+    var username = $(this).attr('name');
+
+    if (interest) {
+      var routeUrl = "addInterest";
+    }
+    else {
+      var routeUrl = "deleteInterest";
+    }

```

Comment:
This if/else could be a lot simpler with a ternary operator: `var routeUrl = interest ? "addInterest" : "deleteInterest;`

Reply:
Fixed.